_dashboards/index.md

```diff
@@ -11,15 +11,42 @@ redirect_from:
+OpenSearch Dashboards is an integrated visualization tool that makes it easy for users to explore their data in OpenSearch. From real-time application monitoring, threat detection, and incident management to personalized search, OpenSearch Dashboards gives you data visualization tools to tell your data story.
 
+Out-of-the-box OpenSearch Dashboards includes several tools for discovering and visualizing data, and the OpenSearch suite provides [plugins](https://opensearch.org/docs/latest/dashboards/install/plugins/) to extend and customize dashboard features and functions. 
 
-## Get started with OpenSearch Dashboards
+# Use cases for real-time search and log analytics at scale 
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+ OpenSearch is powered by the Apache Lucene search library, and it supports a number of search and analytics capabilities, such as k-nearest neighbors (k-NN) search, SQL, anomaly detection, trace analytics, full-text search, and more.  
+
+<table bgcolor="lightgrey" width="600">
+<tbody>
+<tr>
+<td style="text-align: center; vertical-align: top; font-weight: bold;">Visualize log and trace data with interactive log analytics.<img src="{{site.url}}{{site.baseurl}}/images/visualize-log-data.png" alt="Log analytics"/></td>
+<td style="text-align: center; vertical-align: top; font-weight: bold;">Detect and mitigate issues faster with anomaly detection.<img src="{{site.url}}{{site.baseurl}}/images/anomaly-detection.png" alt="Anomaly detection"/></td>
+</tr>
+<tr>
+<td style="text-align: center; vertical-align: top; font-weight: bold;">Diagnose performance issues and reduce application downtime using the observability interface and log monitoring features.<img src="{{site.url}}{{site.baseurl}}/images/observability.png" alt="Observability interface and log monitoring features" /></td>
+<td style="text-align: center; vertical-align: top; font-weight: bold;">Unlock real-time search, monitoring, and analysis of business and operational data.<img src="{{site.url}}{{site.baseurl}}/images/analyzing-data-logs.png" alt="Real-time search, monitoring, and analysis" /></td>
+</tr>
+</tbody>
+</table> 
+
+# Explore the OpenSearch Dashboards playground
+
+Follow these steps to explore the basics of OpenSearch Dashboards:
+
+1.  Go to the [OpenSearch Dashboards playground](https://playground.opensearch.org/app/home).
+2.  Choose **OpenSearch Dashboards > Dashboards > Create dashboard**. 
```

Comment:
On the UI, it says "Dashboard" not "Dashboards". See https://playground.opensearch.org/app/opensearch_dashboards_overview#/. Users can then click on an option to explore the different scenarios. See https://playground.opensearch.org/app/dashboards#/list?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now)). If they click "Create Dashboard", there is nothing there to explore.

Reply:
In the playground, the dashboards are already created for you. You're just viewing what is provided.

Reply:
Deleted "Create dashboard" instruction. User can explore one the three options per step 3.